### DISCUSSION TITLE
Temperaturüberwachung Anker Solix1600

### DISCUSSION THREAD

**Discussion by Hotzi57** [thomluther/anker-solix-api#135] [Q&A] [answered]
Ich würde gern die Temperatur der Solarbank Anker1600 im Homeassitant darstellen, um im Winter entsprechend reagiren zu können. Ich habe den Akku in einer Styropobox installiert und will mal sehen, wie kalt es draußen werden kann, bis er nicht mehr richtig arbeitet. jetzt sind ja 0-55°C angegeben.
In der Anker-App unter dem Punkt "Geräte" wird mit unter Gerätestatus der Ladezustand und die Temperatur angezeigt. Leider finde ich die Temperatur nirgendwo in meiner Homeassitant Integration. Kann mir da jemand weiter hefen?
Vielen Dank.
In Google-Englisch:
I would like to display the temperature of the Anker1600 solar bank in Homeassistant so that I can react accordingly in winter. I have installed the battery in a Styrofoam box and want to see how cold it can get outside before it no longer works properly. It is currently specified as 0-55°C.
In the Anker app under "Devices" the charge level and temperature are displayed under device status. Unfortunately I cannot find the temperature anywhere in my Homeassistant integration. Can anyone help me?
Thank you very much.

**Comment by thomluther** [comment, marked as answer]
Hello @Hotzi57 
I think this question is covered by [HA integration issue 26](https://github.com/thomluther/ha-anker-solix/issues/26)

I recommend to get another temperature sensor that can be added easily to HA (Zigbee or Wifi) and put that into the box as well or attach it to the solarbank case.
Optionally the outside temperature of a weather integration might be a good first indication, but another sensor inside the box is better.

**Comment by Hotzi57** [comment]
Vielen Dank!
